targets/TARGET_RENESAS/TARGET_RZ_A2XX/r_can/src/r_can_rz.c

```diff
@@ -0,0 +1,2758 @@
+/***********************************************************************************************************************
+* DISCLAIMER
+* This software is supplied by Renesas Electronics Corporation and is only intended for use with Renesas products. No
```

Comment:
What license is this, is this permissive? I recall seeing this in older files and our license scanning returned warnings about these files.
Would it be possible to use permissive license for these new files (there will be a new ticket to fix already merged files).

Reply:
Our license check in Travis reporteed only missing SPDX

```
File: r_can_rz_config.h reason: Missing SPDX license identifier
File: r_can_rz_if.h reason: Missing SPDX license identifier
File: r_can_rz.c reason: Missing SPDX license identifier
```

Reply:
Thank you. I added Apache-2.0 license to the pointed files.